gnc/navigator_controller/nodes/mrac_controller.py

```diff
@@ -126,6 +130,22 @@ def set_traj(self, msg):
         self.a_ref = R.dot(np.array([msg.posetwist.acceleration.linear.x, msg.posetwist.acceleration.linear.y, msg.posetwist.acceleration.linear.z]))[:2]
```

Comment:
[nit] Consider using the existing unpackers for pose, posetwist, odom, quaternions, etc